### PR TITLE
docs: re-work foundation pages

### PR DESCRIPTION
**Pull request by hippee-lee**
This PR is a re-working of the foundation content for a core centric world. Pulling some content from the core storybook and adding where it made sense and I could, this should be enough of a MVP that we can build on and enhance with the missing pieces. Listed above are seven tasks that I need to address before the website is production ready. I'm not sure if thats in this PR or a separate PR's. 

# Changes

- move app-layout into angular components
- add layouts content overview, vertical, horizontal, grid and utils
- add design tokens content
- adds a web-component for the website `token-table` and port functions from storybook for generating token metadata

# TODO

0. create foundation content for interaction styles
1. token table css isn't working
2. token table template needs responsiveness
3. i18n content needs updated w for core but not enough ins in storybook so need to use it so I can write it
4. typography content needs to be updated for core with the token-table metadata. What to do with the angular specific stuff - keep and move to angular or toss it?
5. design updates (in Figma) for form controls
6. Navigation content - update for core? move ng specific? toss?

Signed-off-by: Matt Hippely <mhippely@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [x] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by bbogdanov — APPROVED** [review]

**Review by bdryanovski — APPROVED** [review]
LGTM

File: apps/website/foundation/layouts/utilities.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+</div>
+```
+
+</doc-code>
+
+:::
+
+::: component-section-level-one-title
+
+Screen Reader Only
```
Comment (mathisscott) [inline apps/website/foundation/layouts/utilities.md:57]: No header level here? I would expect `##` or `###`

**Review by mathisscott — CHANGES_REQUESTED** [review]

File: apps/website/foundation/layouts/utilities.md (hunk 1 above)
Comment (hippee-lee) [reply apps/website/foundation/layouts/utilities.md:57]: Too much egg nog 😝 ...

**Comment by hippee-lee** [comment]
Closing this as a POC instead of a proposed change for the v5 release. 
I'll keep the branch so it could be used as a starting point if needed later on.

**Pull request closed without merging**